aten/src/ATen/core/dispatch/DispatchTable.h

```diff
@@ -165,6 +175,7 @@ class DispatchTable final {
 
   const KernelFunction* lookup(DispatchKey dispatchKey) const {
     auto& slot = kernels_[dispatchKey];
+    // TODO: this condition shouldn't be necessary
```

Comment:
This is just a general TODO I noticed when writing the PR.

Reply:
can you give more context? "shouldn't be necessary" as in "I expect this still works if I remove it today" or as in "we should refactor things so it isn't necessary anymore"? If the latter, give more details on what refactoring you have in mind.

Reply:
We should engineer things so that the slot is always valid, and if we return the slot for something initialized, we get a "default" value that results in the correct result in dispatching.

Reply:
Can you add that as a comment to the todo?